edr_plugin/gui/query_tools.py

```diff
@@ -523,9 +523,22 @@ def setup_data_query_tool(self):
         else:
             self.output_crs = QgsCoordinateReferenceSystem.fromOgcWmsCrs(crs_name)
         corridor_query_data = self.edr_dialog.query_cbo.currentData()
-        width_units = corridor_query_data["link"]["variables"]["width-units"]
+        variables = corridor_query_data["link"]["variables"]
+        if "width-units" not in variables:
+            self.edr_dialog.plugin.communication.show_error(
+                "Collection does not have information about `width-units`. The collection is not up with specification."
```

Comment:
Please can this text be changed as follows: "The collection does not have information about `width-units` so is not compatible with the OGC API specification."